v63003/u_inref/gtm4212.csh

```diff
@@ -0,0 +1,42 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+#
+
+$ydb_dist/mumps -run path230^gtm4212
```

Comment:
Same comment as before. The below code can also be folded into a loop that way we avoid repeating each type of line 4 times.